commands/comment.go

```diff
@@ -58,7 +58,7 @@ func checkCommentLocation(repo repository.Repo, commit, file string, line uint)
 		return err
 	}
 	lines := strings.Split(contents, "\n")
-	if line >= uint(len(lines)) {
+	if line < 1 || line > uint(len(lines)) {
```

Comment:
@mattn, sorry if I was not clear enough.

Zero is a valid value to pass to the `-l` flag. So, for instance:

    git appraise comment -m test -f README.md -l 0 b2a042b3773555f73aef5a840289b3ca110612de

is allowed, and is equivalent to (note the absence of the `-l` flag):

    git appraise comment -m test -f README.md b2a042b3773555f73aef5a840289b3ca110612de

When a user does that, they are commenting on the entire file rather than a single line in the file.

The `checkCommentLocation` method is called any time that the `-f` flag is specified, so if we added the lower bound check (`line < 1...`), then these whole-file comments would no longer be allowed.

The `commentLine` variable is defined to be an unsigned int, and 0 is a valid value, so we do not need to perform any lower-bounds checking for it.